pkg/asset/cluster/tfvar.go

```diff
@@ -0,0 +1,86 @@
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/config"
+)
+
+const tfvarFilename = "terraform.tfvar"
+
+// TerraformVariables depends on InstallConfig and
+// Ignition to generate the terrafor.tfvars.
+type TerraformVariables struct {
+	// The root directory of the generated assets.
+	rootDir string
+	// The Assets that this tfvar file depends.
+	installConfig     asset.Asset
+	bootstrapIgnition asset.Asset
+	masterIgnition    asset.Asset
+	workerIgnition    asset.Asset
+}
+
+var _ asset.Asset = (*TerraformVariables)(nil)
+
+// Dependencies returns the dependency of the TerraformVariable
+func (t *TerraformVariables) Dependencies() []asset.Asset {
+	return []asset.Asset{t.installConfig, t.bootstrapIgnition, t.masterIgnition, t.workerIgnition}
+}
+
+// Generate generates the terraform.tfvar file.
+func (t *TerraformVariables) Generate(parents map[asset.Asset]*asset.State) (*asset.State, error) {
+	state, ok := parents[t.installConfig]
+	if !ok {
+		return nil, fmt.Errorf("failed to get install config state in the parent asset states")
+	}
+
+	var installConfig types.InstallConfig
+	if err := yaml.Unmarshal(state.Contents[0].Data, &installConfig); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal install config: %v", err)
+	}
+
+	state, ok = parents[t.bootstrapIgnition]
+	if !ok {
+		return nil, fmt.Errorf("failed to get the bootstrap ignition state in the parent asset states")
+	}
+	bootstrapIgn := string(state.Contents[0].Data)
+
+	state, ok = parents[t.masterIgnition]
+	if !ok {
+		return nil, fmt.Errorf("failed to get the master ignition state in the parent asset states")
+	}
+	var masterIgns []string
+	for _, st := range state.Contents {
+		masterIgns = append(masterIgns, string(st.Data))
+	}
+
+	state, ok = parents[t.workerIgnition]
```

Comment:
I'd thought worker ignition files would be obsolete since #119, although it looks like [we still have some local code for them][1].  Do you know who's going to be responsible for them long-term?

[1]: https://github.com/openshift/installer/pull/119/files#r218293765

Reply:
Talked with @abhinavdahiya , looks like we still need to pass the worker ignition to the operator.